recipe/conda_build_config.yaml

```diff
@@ -6,3 +6,5 @@ MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
   - "10.14"                # [osx and x86_64]
 CONDA_BUILD_SYSROOT:       # [osx and x86_64]
   - /opt/MacOSX10.10.sdk   # [osx and x86_64]
+libtiff:
+  - 4.5.1
```

Comment:
```suggestion
```
This may cause issues in future rebuilds, if packagers fail to notice this. Besides, our cbc pins to 4.5.0, which will pull the latest libtiff at runtime.

Reply:
Removed